src/Microsoft.VisualBasic/tests/Microsoft.VisualBasic.Tests.builds

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" >
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="Microsoft.VisualBasic.Tests.csproj" />
```

Comment:
Just as a reminder of our chat online, this should be disabled for non-windows platforms.